DfE.FindInformationAcademiesTrusts/Services/ExportService.cs

```diff
@@ -0,0 +1,100 @@
+﻿using ClosedXML.Excel;
+using DfE.FindInformationAcademiesTrusts.Data;
+using DfE.FindInformationAcademiesTrusts.Pages;
+
+namespace DfE.FindInformationAcademiesTrusts.Services
+{
+    public interface IExportService
+    {
+        byte[] ExportAcademiesToSpreadsheetUsingProvider(Data.Trust? trust, Trust.TrustSummaryServiceModel? trustSummary);
+    }
+
+    public class ExportService : IExportService
+    {
+
+        public byte[] ExportAcademiesToSpreadsheetUsingProvider(Data.Trust? trust, Trust.TrustSummaryServiceModel? trustSummary)
```

Comment:
Under current usage, trust can never be null so we should use a non-nullable type (and remove the conditional access further down)

Reply:
Nice, done